### DISCUSSION TITLE
Storage Bucket Endpoints

### DISCUSSION THREAD

**Discussion by nithin-pb** [supabase/supabase#2157] [Questions] [answered]
Hi, I would like to know if there any endpoints are available by default for superbase buckets?

**Comment by inian** [comment, marked as answer]
Hi you can access the storage api endpoints at yourprojectref.supabase.co/storage/v1 and the docs are [here](https://supabase.github.io/storage-api/#/)

**Reply by nithin-pb** [reply]
Thanks @inian
